Obsidian/Commands/MainCommandModule.cs

```diff
@@ -224,7 +224,7 @@ public async Task GamemodeAsync(CommandContext Context, [Remaining] string args_
         var args = args_.Contains(" ") ? args_.Split(" ").ToList() : new List<string> { args_ };
         if (args.Count == 1)
         {
-            if (args[0].ToLower() == "creative" || args[0].ToLower() == "survival" || args[0].ToLower() == "spectator" || args[0].ToLower() == "adventure")
+            if (new[] { "creative", "survival", "spectator", "adventure" }.Contains(args[0].ToLowerInvariant()))
```

Comment:
Allocating an array for each call is not ideal, but it's still better than what was there previously.